src/components/security-code-expired/security-code-expired-controller.ts

```diff
@@ -0,0 +1,9 @@
+import { Request, Response } from "express";
+
+export function securityCodeExpiredGet(req: Request, res: Response): void {
+  res.render("security-code-expired/index.njk");
+}
+
+export function securityCodeExpiredPost(req: Request, res: Response): void {
```

Comment:
do we need this post method?

Reply:
removed pending clarification on content + behaviour